sys/dev/ath/ath_hal/ar9002/ar9271_attach.c

```diff
@@ -493,12 +495,37 @@ ar9271FillCapabilityInfo(struct ath_hal *ah)
 static const char*
 ar9271Probe(uint16_t vendorid, uint16_t devid)
 {
-	//TODO: add vendor AR9271 vendor
-	// if (vendorid == ATHEROS_VENDOR_ID && devid == AR9285_DEVID_PCIE)
-	// 	return "Atheros 9285";
-	// if (vendorid == ATHEROS_VENDOR_ID && (devid == AR2427_DEVID_PCIE))
-	// 	return "Atheros 2427";
-
+	if (vendorid == USB_VENDOR_ATHEROS2) {
+		if (devid == USB_PRODUCT_ATHEROS2_AR9271_1 ||
+			devid == USB_PRODUCT_ATHEROS2_AR9271_2 ||
```

Comment:
Here and in other places: 
- alignment
- I'm not sure if syntax with additional brackets for complex expressions is preferred i.e.: 
         
        if ((devid == USB_PRODUCT_ATHEROS2_AR9271_1) ||
	    (devid == USB_PRODUCT_ATHEROS2_AR9271_2) ||

Reply:
I will fix it.

Reply:
Done